src/java.base/share/classes/sun/security/util/resources/security_de.properties

```diff
@@ -43,7 +43,7 @@ provided.null.OID.map=Null-OID-Zuordnung angegeben
 NEWLINE=\n
 invalid.null.action.provided=Ungültige Nullaktion angegeben
 invalid.null.Class.provided=Ungültige Nullklasse angegeben
-Subject.=Subjekt:\n
```

Comment:
This change reverts what was previously changed in the last l10n drop. There was a request to change word "Subject" to "Subjekt" at the time, and this was manually adjusted. This issue was brought up to the translation team but was ultimately rejected. So, to align with upstream, I will leave this as is.